contracts/VotingMachines/GenesisProtocol.sol

```diff
@@ -83,6 +85,11 @@ contract GenesisProtocol is IntVoteInterface,UniversalScheme {
         mapping(address=>Staker) stakers;
     }
 
+    struct ExpieredProposal {
```

Comment:
This struct seems to never be used. Does it have any future purpose? If not, I guess it should be removed.

Reply:
Removed at c9c7256